movingpandas/io.py

```diff
@@ -1,10 +1,146 @@
 import json
+from typing import Callable, Dict
 
+from geopandas import GeoDataFrame
 from pandas import DataFrame
 
 from movingpandas import Trajectory
 
 
+def gdf_to_mf_json(
+    gdf: GeoDataFrame,
+    traj_id_property: str,
```

Comment:
Please use "column" instead of "property" to make it clear that these are referring to columns of the GeoDataFrame

Reply:
Done!